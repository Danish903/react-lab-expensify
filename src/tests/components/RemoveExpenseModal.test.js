import React from 'react';
import { shallow } from 'enzyme';
import RemoveExpenseModal from '../../components/RemoveExpenseModal';

test('should render Remove Modal correctly', () => {
  const wrapper = shallow(
    <RemoveExpenseModal
      modalIsOpen
      handleConfirmRemove={() => {}}
      handleCancelRemove={() => {}}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
