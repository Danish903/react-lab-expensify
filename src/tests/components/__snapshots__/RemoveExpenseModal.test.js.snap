// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Remove Modal correctly 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="modal"
  closeTimeoutMS={200}
  contentLabel="Remove Confirmation"
  isOpen={true}
  onRequestClose={[Function]}
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
>
  <h3
    className="modal__title"
  >
    Remove Confirmation
  </h3>
  <p
    className="modal__body"
  >
    Remove this expense?
  </p>
  <div
    className="modal__button"
  >
    <button
      className="button"
      onClick={[Function]}
    >
      Confirm
    </button>
    <button
      className="button button--secondary"
      onClick={[Function]}
    >
      Cancel
    </button>
  </div>
</Modal>
`;
